utils/lib/rr_graph/channel.py

```diff
@@ -243,318 +268,516 @@ def __str__(self):
 
 
 # Nice short alias..
-C = Channel
+T = Track
 
 
 class ChannelGrid(dict):
+    '''
+    Functionality:
+    -Manages single type of track (either chanx or chany)
+    -Channel width along grid
+    -Manages track allocation within channels
+    -A track allocator
+
+    dict is indexed by Pos() objects
+    This returns a list indicaitng all the tracks at that position
+    '''
     def __init__(self, size, chan_type):
+        '''
+        size: Size representing tile grid width/height
+        chan_type: of Channels.Type
+        '''
         self.chan_type = chan_type
         self.size = Size(*size)
 
-        for x in range(0, self.x):
-            for y in range(0, self.y):
+        for x in range(0, self.width):
+            for y in range(0, self.height):
                 self[Pos(x,y)] = []
 
     @property
-    def x(self):
-        return self.size.x
+    def width(self):
+        """Grid width
```

Comment:
Maybe something like "Number of channels (as opposed to the width a an individual channel?)"